src/inverseRobot.js

```diff
@@ -25,6 +25,16 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const result = {};
+
+  for (const key in robot) {
```

Comment:
You could also use Object.entries(robot) with destructuring here

```
  for (const [key, value] of Object.entries(robot)) {
   // your code
  }
```

Reply:
Yeah, as one of the ways to iterate through object